test/account_test.rb

```diff
@@ -86,13 +86,14 @@ def app
       stub_heroku_api
       post "/account/password/reset", email: "kerry@heroku.com"
       assert_equal 200, last_response.status
+      refute last_request.env['x-rack.flash']['notice'].nil?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.